pkg/apis/agones/v1/gameserver_test.go

```diff
@@ -152,12 +155,14 @@ func TestGameServerApplyDefaults(t *testing.T) {
 					InitialDelaySeconds: 11,
 					PeriodSeconds:       12,
 				},
+				sdkServerLogLevel: "Info",
```

Comment:
this test is "defaults are already set" so the log level should be set in the gameserver as well as in the expected result.

Reply:
Done